### PR TITLE
Cleanup/misc cleanups

### PR DESCRIPTION
**Pull request by smoser**

### CODE REVIEW

File: layers/build-krd/dracut/soci/README.md [hunk 1]
```diff
@@ -41,3 +41,11 @@ and metadata.yaml.signed signed with the private key for manifestCert.pem.
 You can mount that with:
 
     root=soci:name=rootfs-soci,dev=LABEL=oci-data,path=oci-data
+
+## Notes
+ * This module's install does *not* copy in all it's dependencies.  This is
```
Comment (hallyn) [inline layers/build-krd/dracut/soci/README.md:46]: it's is a contraction of "it is".  You want "its" here.

File: layers/build-krd/dracut/soci/soci-settled.sh [hunk 2]
```diff
@@ -245,17 +241,32 @@ soci_udev_settled() {
...
         done
 
-        cp "/pcr7data.cpio" "$rootd/"
-        ( mkdir -p "$rootd/pcr7data"; cd "$rootd/pcr7data"; cpio -id < /pcr7data.cpio )
+        # copy the pcr7data to persist past initramfs
+        mkdir -p /run/machine && cp -r /pcr7data /run/machine/ || {
+            soci_die "Unable to copy pcr7data to /run/machine"
+            return 1
+        }
+        ## FIXME - I don't like the polution of target's /
```
Comment (hallyn) [inline layers/build-krd/dracut/soci/soci-settled.sh:255]: Should we put it at /factory/secure/pcr7data ?

Comment (hallyn) [reply layers/build-krd/dracut/soci/soci-settled.sh:255]: BTW you did raise a good point earlier - pcr7data in initramfs has problems, and we should have a partition with that data, which gets updated.  (In particular, installing an older ISO on newer firmware can break when pcr7 changes).

**Review by hallyn — APPROVED** [review]

File: layers/build-krd/dracut/soci/soci-settled.sh (hunk 2 above)
Comment (smoser) [reply layers/build-krd/dracut/soci/soci-settled.sh:255]: > Should we put it at /factory/secure/pcr7data ?

i dont know. it didn't seem like 'factory' was right. But I guess that is fine for now.

**Pull request merged**